index.html

```diff
@@ -6,12 +6,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
     <title>CodersCamp2020 | Pokemon Fight</title>
-    <link rel="stylesheet" href=""/>
+    <link rel="stylesheet" href="styles/app.css">
 </head>
 
 <body>
 
 
+    <!-- <div id="loading-view">
+        <p>Please wait a bit, we are looking for the Poké Flute to wake up Snorlax to fight.</p>
+        <img id="zzz" src="assets/zzz.png" alt="zzz">
+        <img src="assets/snorlax.png" alt="sleeping snorlax">
+    </div> -->
+
+    <!-- <template id="starting-page-template"> -->
+        <div id="starting-page">
+            <div id="header-with-enter-name-manu">
+                <div class="headerWrapper">
+                    <p class="gameTitle">Pokemon Fight</p>
+                    <p class="startInstruction">Enter yours names to start the Fight!</p>
+                    <div id="enter-name-area">
+                        <input id="enter-player1-name" class="playerNameInput" value="Player 1"/>
+                        <input id="enter-player2-name" class="playerNameInput" value="Player 2"/>
+                    </div>
+                </div>
+            </div>
+            <div id="start-game-border">
+                <div id="start-game-bg1">
+                    <div id="start-game-bg2">
+                        <div id="start-game-btn"></div>
+                    </div>
+                </div>
+            </div>
+            <div id="info-menu">
+                <ul id=select-info-menu>
+                    <li id="how-to-play-btn" class="start-page-button">How to play</li>
+                    <li id="about-btn" class="start-page-button">About</li>
+                    <li id="authors-btn" class="start-page-button">Authors</li>
+                </ul>
+            </div>
+        </div>
+    <!-- </template> -->
+
+    <footer>
+        TypeScript Mangos project for CodersCamp2020
+    </footer>
 
     <script src="dist/index.js"></script>
```

Comment:
shouldn't it be "src/index.ts"?

Reply:
As I understand index.ts will be compiled to index.js and after that it could be used in html file.